### PR TITLE
Added log2 instance method to Paddle Tensor #16419

### LINKED ISSUE #16419: log2

### PR DESCRIPTION
**Pull request by farazamjad**
close #16419 was staled. New PR created

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @farazamjad 

Thanks for the PR😊
Would you add unit tests of the implemented function

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py [hunk 1]
```diff
@@ -2264,16 +2264,16 @@ def test_paddle_conj(
     )
 
 
-# floor_
+# log2
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:2267]: Seems like you're entirely removing `floor_` tests

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey,
Would you explain why you're renaming the existing unit tests instead of implementing your own🙂

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py (hunk 1 above)
Comment (farazamjad) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:2267]: Sorry I wasn't renaming it I guess when I created the PR floor_ was not implemented in test. And my IDE automatically sync with latest repo.I wasn't aware of it I removed it and thought I did something. But now I have added floor_ you can review again and let me know.

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 2]
```diff
@@ -489,12 +489,19 @@ def floor_divide(self, y, name=None):
     def conj(self, name=None):
         return ivy.conj(self._ivy_array)
 
+
+
+    @with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+    def log2(self, name=None):
+        return ivy.log2(self._ivy_array)
+
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/paddle/tensor/tensor.py:497]:
```suggestion



```

**Pull request merged**